### ISSUE TITLE
Getting error saying their signature doesnot match in lstm

### ISSUE THREAD

**Issue by Ankan1998** [alleveenstra/attentionocr#16]
Epoch 000 / 001 : : 1it [00:02,  2.66s/it, training loss=4.0050, iterations=1]
2021-01-25 17:52:40.330843: W tensorflow/core/grappler/optimizers/implementation_selector.cc:310] Skipping optimization due to error while loading function librarie
s: Invalid argument: Functions '__inference_standard_lstm_6471' and '__inference_standard_lstm_6471_specialized_for_lstm_1_5_StatefulPartitionedCall_at___inference_
keras_scratch_graph_71971' both implement 'lstm_287408bc-d5fd-45ea-a08f-389dac98439a' but their signatures do not match.
2021-01-25 17:52:55.098128: W tensorflow/core/grappler/optimizers/implementation_selector.cc:310] Skipping optimization due to error while loading function librarie
s: Invalid argument: Functions '__inference_standard_lstm_43101' and '__inference_standard_lstm_43101_specialized_for_lstm_1_64_StatefulPartitionedCall_at___inferen
ce_keras_scratch_graph_77144' both implement 'lstm_cfe8d53e-9ee3-42c4-9653-ac8c91a62f7f' but their signatures do not match.

This is the error, I am getting. Any help is highly appreciated
